src/main/java/mil/dds/anet/config/AnetConfiguration.java

```diff
@@ -234,4 +322,24 @@ public void setSslTrust(String sslTrust) {
     }
   }
 
+  @SuppressWarnings("unchecked")
+  public String loadVersion() {
+    try (final InputStream inputStream =
```

Comment:
Can we store the correct anet version in the anet jar manifest and then retrieve it through [getImplementationVersion](https://docs.oracle.com/javase/8/docs/api/java/lang/Package.html#getImplementationVersion--
)

Reply:
May I open a debt issue for this CR ?